src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/DirectorsReportApprovalController.java

```diff
@@ -69,8 +69,19 @@ public String getDirectorsReportApproval(@PathVariable String companyNumber,
                     directorsReportApprovalService.getDirectorsReportApproval(transactionId, companyAccountsId);
 
             List<String> approverOptions =
-                    Arrays.stream(directorService.getAllDirectors(transactionId, companyAccountsId))
-                            .filter(d -> d.getResignationDate() == null)
+                    Arrays.stream(directorService.getAllDirectors(transactionId, companyAccountsId)).map(
```

Comment:
I have only one Q here, but it may be a bit larger than this PR. What is the largest size of reply for getAllDirectors? Would it (should it) ever be paged?

Reply:
Max is 20 so we never worry about paging!